plugins/ScheduledReports/ScheduledReports.php

```diff
@@ -207,6 +200,18 @@ public function getReportMetadata(&$reportMetadata, $reportType, $idSite)
         $reportMetadata = $filteredReportMetadata;
     }
 
+    private function shouldReportBeIgnored($reportMetadata)
+    {
+        // Some plugins may not define report categories
+        if(empty($reportMetadata['category'])) {
+            return false;
+        }
+
+        $isAPI = $reportMetadata['category'] == 'API';
+        $isMultiSitesGetOne = $reportMetadata['category'] == Piwik::translate('General_MultiSitesSummary') && $reportMetadata['name'] == Piwik::translate('General_SingleWebsitesDashboard');
```

Comment:
Note I haven't changed the logic, only refactored, so it wouldn't regress. Maybe it can be improved but not in this PR I suppose...